### ISSUE TITLE
Improve design of creator status on the profile page

### ISSUE THREAD

**Issue by Dumzy006** [unstructuredstudio/zubhub#897]
Hello @srish I noticed on my profile page that the creator status looks more like a an active button which shouldn't be so
Here is a screenshot of how it currently looks.
![image](https://github.com/unstructuredstudio/zubhub/assets/136137889/8037059d-cae2-4a22-8938-02738bcea196)

Here is another screenshot of an improved version on how it should look like. What do you think?
![image](https://github.com/unstructuredstudio/zubhub/assets/136137889/075702ad-2eb8-4f39-983f-c2585c33d76f)

**Comment by aqsaaqeel** [comment]
Ooh, I like where you're going with this, maybe we can have the `creator status` next to the username because we have plenty of dead space on the right that is not being utilized yet? what are your thoughts? @Dumzy006

**Comment by Dumzy006** [comment]
Hello @aqsaaqeel 
I see where you are coming from, so I made a design based on what you said
See screenshots below
![image](https://github.com/unstructuredstudio/zubhub/assets/136137889/889cdcd4-1291-44bc-9662-32f10810650e)
![image](https://github.com/unstructuredstudio/zubhub/assets/136137889/889cdcd4-1291-44bc-9662-32f10810650e)

**Comment by brrkrmn** [comment]
@Dumzy006 I think it looks more compact and aligned under the username 🤔

**Comment by Dumzy006** [comment]
> @Dumzy006 I think it looks more compact and aligned under the username 🤔

I feel the first one is better @brrkrmn @aqsaaqeel which is this

![image](https://github.com/unstructuredstudio/zubhub/assets/136137889/9c51eba1-179f-4d9c-b054-e655db61cd54)

**Comment by brrkrmn** [comment]
@Dumzy006 yes I agree. I'll start with this then

**Comment by brrkrmn** [comment]
@Dumzy006 would you mind checking the PR for verification?

**Comment by Dumzy006** [comment]
@brrkrmn It looks good

**Comment by tuxology** [comment]
Valid point @Dumzy006 but I think we need better ideas. One good thing about the chip style for these "tags" is that its clearly visible. We plan to add more such tags eventually (Staff, Educator etc) We can possibly use a gray chip at the very least here. What do you think?

**Comment by Dumzy006** [comment]
Hello @tuxology 
I made an update using the gray for the tag
See screenshot below

![image](https://github.com/unstructuredstudio/zubhub/assets/136137889/c865d827-ea6a-49ea-beb8-8372752f1ba6)
This is for when there are more than one tags
![image](https://github.com/unstructuredstudio/zubhub/assets/136137889/90a65d29-acdf-4305-8145-c62bb4252e32)

**Comment by DonPresh** [comment]
@Dumzy006 , I think these gray tags look like disabled buttons. Maybe reduce the padding to make it look more like a tag than a button.

**Comment by Dumzy006** [comment]
Alright @DonPresh 
Updated that in the design above

**Comment by DonPresh** [comment]
A lot better. Also, I know the mentor said maybe gray, but don't you think this is a better contrast than the gray? @Dumzy006 
<img width="395" alt="Screenshot 2023-10-18 at 12 42 39" src="https://github.com/unstructuredstudio/zubhub/assets/100778015/515f9e53-936e-4e08-bdb2-bece82b6584a">

or even this?
<img width="408" alt="Screenshot 2023-10-18 at 12 45 03" src="https://github.com/unstructuredstudio/zubhub/assets/100778015/616ee84c-9521-4629-b312-8c9cf2abb485">

Maybe align to the side instead of center. What do you think?

**Comment by Dumzy006** [comment]
It is actually but clickable items likes projects, followers etc have these colours. That was why I didn't use it @DonPresh

**Comment by Dumzy006** [comment]
I just checked and this is the colour currently used on the site to represent tags @DonPresh 
See screenshot below

![image](https://github.com/unstructuredstudio/zubhub/assets/136137889/3c270db1-ec26-45cd-97d6-45434c1a0f17)


So I  came up with this
![image](https://github.com/unstructuredstudio/zubhub/assets/136137889/3317179e-6da8-489f-ab22-66271717811d)

**Comment by Temabo** [comment]
> A lot better. Also, I know the mentor said maybe gray, but don't you think this is a better contrast than the gray? @Dumzy006 <img alt="Screenshot 2023-10-18 at 12 42 39" width="395" src="https://user-images.githubusercontent.com/100778015/276233962-515f9e53-936e-4e08-bdb2-bece82b6584a.png">
> 
> or even this? <img alt="Screenshot 2023-10-18 at 12 45 03" width="408" src="https://user-images.githubusercontent.com/100778015/276234624-616ee84c-9521-4629-b312-8c9cf2abb485.png">
> 
> Maybe align to the side instead of center. What do you think?

I agree with this and I like both iterations, they look like tags. The filled gray tags look more like inactive buttons, maybe it's the shade, but the contrast looks a bit off with the white text. @Dumzy006

**Comment by Temabo** [comment]
> I just checked and this is the colour currently used on the site to represent tags @DonPresh See screenshot below
> 
> ![image](https://user-images.githubusercontent.com/136137889/276237035-3c270db1-ec26-45cd-97d6-45434c1a0f17.png)
> 
> So I came up with this ![image](https://user-images.githubusercontent.com/136137889/276236537-3317179e-6da8-489f-ab22-66271717811d.png)

looks good!

**Comment by Dumzy006** [comment]
I feel we should maintain consistency with what is on the site except we want to totally propose or have a new colour for tags. What do you think ? @DonPresh @temabo

**Comment by DonPresh** [comment]
> I feel we should maintain consistency with what is on the site except we want to totally propose or have a new colour for tags. What do you think ? @DonPresh @Temabo

I get what you are saying. We can design with the consistency of what they have, while simultaneously proposing what can be done to improve the UI. 
For instance we can show a screen of "This is how I think this should work" maintaining consistency.
Then do a follow up iteration of how we think the experience will be better.

It's all part of the contribution. I don't know if i'm clear @Dumzy006 @Temabo

**Comment by Temabo** [comment]
@D

> > I feel we should maintain consistency with what is on the site except we want to totally propose or have a new colour for tags. What do you think ? @DonPresh @Temabo
> 
> I get what you are saying. We can design with the consistency of what they have, while simultaneously proposing what can be done to improve the UI. For instance we can show a screen of "This is how I think this should work" maintaining consistency. Then do a follow up iteration of how we think the experience will be better.
> 
> It's all part of the contribution. I don't know if i'm clear @Dumzy006 @Temabo

I get you @DonPresh Although we can see there is an existing design for the tag, we can still tweak it to make it better, The current design still has some contrast issues. I feel we can adjust the current colour so it's better. @Dumzy006

**Comment by DonPresh** [comment]
> @d
> 
> > > I feel we should maintain consistency with what is on the site except we want to totally propose or have a new colour for tags. What do you think ? @DonPresh @Temabo
> > 
> > 
> > I get what you are saying. We can design with the consistency of what they have, while simultaneously proposing what can be done to improve the UI. For instance we can show a screen of "This is how I think this should work" maintaining consistency. Then do a follow up iteration of how we think the experience will be better.
> > It's all part of the contribution. I don't know if i'm clear @Dumzy006 @Temabo
> 
> I get you @DonPresh Although we can see there is an existing design for the tag, we can still tweak it to make it better, The current design still has some contrast issues. I feel we can adjust the current colour so it's better. @Dumzy006

Sure sure.

**Comment by Boluwatifey1** [comment]
Hi @Dumzy006 @Temabo @tuxology @DonPresh Since we have agreed to design the tags like in #917 I came up with this design for labels
![profile edit (1)](https://github.com/unstructuredstudio/zubhub/assets/127180289/92b7e54c-4527-4c31-98fd-948e63895e80)

what do you guys think?

**Comment by srish** [comment]
Let's wait for https://github.com/unstructuredstudio/zubhub/issues/917 to mature a bit before we make a final call on the design developed to address this issue.

**Comment by mehreeee** [comment]
Hi @srish, @tuxology  I think the grey tags look better here's my suggestion.
![Frame 1000002845](https://github.com/unstructuredstudio/zubhub/assets/105458062/618aaf1a-aa4d-47de-a200-e7f481f64375)

**Comment by Dumzy006** [comment]
Hello @mehreeee, nice suggestion
But as @srish said we have to wait for  https://github.com/unstructuredstudio/zubhub/issues/917 to be agreed upon. So that we can use the same design for tags there to update this
